src/app/shared/user.service.ts

```diff
@@ -71,14 +71,15 @@ export class UserService {
   endSessionLog() {
     if (this.sessionId === undefined) {
       return this.couchService.post(this.logsDb + '/_find', findDocuments(
-              { 'user': this.get().name, 'logout_time': 0 },
+              { 'user': this.get().name },
               [ '_id', '_rev', 'login_time' ],
-              1
+              [ { '_id': 'desc' } ]

```

Comment:
_id will not be generated such that it will be larger as time pass.
So we could use last login_time instead

Reply:
Yes, but couch db is not allowing me to do it. 
![screen shot 2018-02-13 at 9 17 31 am](https://user-images.githubusercontent.com/7495421/36132351-9e56c690-106e-11e8-9447-c4a5bc4db961.png)